tests/dummy/app/styles/app.css

```diff
@@ -32,6 +32,27 @@ h6:last-child
   margin-bottom: 0;
 }
 
+.height--400 {
+  height: 400px;
+}
+
+.padding--8 {
+  padding: 8px;
+}
+
+.info {
+  font-size: 12px;
+  color: #2f7ada;
```

Comment:
can we have the color stored in a variable

Reply:
This is a plain css file and not a preprocessor like sass/less. Of course, we do have css variables, but its hard to maintain and concerned about the backward-compatibility. Moreover this is under `test/` folder which is to test the addon during development phase and will not have any impact on the consuming applications.